recipes/libsgp4/all/CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+cmake_minimum_required(VERSION 3.4.3)
```

Comment:
Why don't you use https://github.com/dnwrnr/sgp4/blob/master/CMakeLists.txt instead. Or, apply the required patches if it's broken.

Reply:
It's really broken. But yes I can patch CMake files and call build on sgp4 target (maybe better for maintanability, hopping better CMakeLists from upstream in new versions). And I've made a mistake, public headers are copied in include/SGP4, not include/libsgp4.

Reply:
Nice! thanks!